sunpy/instr/lyra.py

```diff
@@ -630,10 +634,10 @@ def split_series_using_lytaf(timearray, data, lytaf):
     mask = np.ones(n)
     el = len(lytaf)
 
-    # make the input time array a list of datetime objects
-    datetime_array = []
+    # make the input time array a list of Time objects
+    time_array = []
     for tim in timearray:
-        datetime_array.append(parse_time(tim))
+        time_array.append(parse_time(tim))
```

Comment:
When I used a `Time` array instead of list of `Time`s it was giving weird errors which I could not track down at the time.

I vote for list comprehension.